app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,16 @@
   <div class="topnav">
     <%=link_to "All Pets", "/pets"%>
     <%=link_to "All Shelters", "/shelters"%>
+    <%=link_to "Favorites: #{pluralize(favorites.total_count, "pet")}", '/favorites' %>
   </div>
 
-
   <body>
+    <% flash.each do |type, msg| %>
+      <section class=<%= type %>>
+        <p><font size="5" color="red"><%= msg %></font></p>
+      </section>
+    <% end %>
+
```

Comment:
moved flash to application view

Reply:
Like this decision. Removing excess code from all other views.